File: algorithm/typescript/BOJ11720.ts
// problem : https://www.acmicpc.net/problem/11720

import fs = require("fs");

const input: string[] = fs.readFileSync("/dev/stdin").toString().split("\n");
const N: number = Number(input[0]);
const str: string = input[1];

let result: number = 0;

for (let i = 0; i < N; i++) {
  result += Number(str.charAt(i));
}

console.log(result);

File: algorithm/java/src/BOJ11720.java
// problem : https://www.acmicpc.net/problem/11720

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class BOJ11720 {
    static int N;
    static int result = 0;

    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        N = Integer.parseInt(br.readLine());
        String str = br.readLine();

        for (int i = 0; i < str.length(); i++) {
            result += str.charAt(i) - '0';
        }

        System.out.println(result);
    }
}
